docker/policyconfiguration/naming_test.go

```diff
@@ -49,7 +50,12 @@ func TestDockerReference(t *testing.T) {
 			moreSpecific := identity
 			for i := range expectedNS {
 				assert.Equal(t, ns[i], expectedNS[i], fmt.Sprintf("%s item %d", fullInput, i))
-				assert.True(t, strings.HasPrefix(moreSpecific, ns[i]))
+				// Check for subdomain matches if wildcard present
```

Comment:
(Nit: This comment really belongs _inside_ the `if` branch. I’m not sure that it is very necessary either way.

Here, a comment to the effect of “verify that `expectedNS` is ordered from most specific to least specific” might be useful to explain the _purpose_ of the code, and that might sufficiently explain thy the subdomain does what it does.)